Sources/X509/RDNAttribute.swift

```diff
@@ -132,8 +229,128 @@ extension RelativeDistinguishedName.Attribute: DERImplicitlyTaggable {
+            ///       utf8String        UTF8String      (SIZE (1..ub-state-name)),
+            ///       bmpString         BMPString       (SIZE (1..ub-state-name)) }
+            ///
+            ///
+            /// -- Naming attributes of type X520OrganizationName
+            ///
+            /// id-at-organizationName  AttributeType ::= { id-at 10 }
+            ///
+            /// -- Naming attributes of type X520OrganizationName:
+            /// --   X520OrganizationName ::=
+            /// --          DirectoryName (SIZE (1..ub-organization-name))
+            /// --
+            /// -- Expanded to avoid parameterized type:
+            /// X520OrganizationName ::= CHOICE {
+            ///       teletexString     TeletexString
+            ///                           (SIZE (1..ub-organization-name)),
+            ///       printableString   PrintableString
+            ///                           (SIZE (1..ub-organization-name)),
+            ///       universalString   UniversalString
+            ///                           (SIZE (1..ub-organization-name)),
+            ///       utf8String        UTF8String
+            ///                           (SIZE (1..ub-organization-name)),
+            ///       bmpString         BMPString
+            ///                           (SIZE (1..ub-organization-name))  }
+            ///
+            ///
+            /// id-at-organizationalUnitName AttributeType ::= { id-at 11 }
+            ///
+            /// -- Naming attributes of type X520OrganizationalUnitName:
+            /// --   X520OrganizationalUnitName ::=
+            /// --          DirectoryName (SIZE (1..ub-organizational-unit-name))
+            /// --
+            /// -- Expanded to avoid parameterized type:
+            /// X520OrganizationalUnitName ::= CHOICE {
+            ///       teletexString     TeletexString
+            ///                           (SIZE (1..ub-organizational-unit-name)),
+            ///       printableString   PrintableString
+            ///                           (SIZE (1..ub-organizational-unit-name)),
+            ///       universalString   UniversalString
+            ///                           (SIZE (1..ub-organizational-unit-name)),
+            ///       utf8String        UTF8String
+            ///                           (SIZE (1..ub-organizational-unit-name)),
+            ///       bmpString         BMPString
+            ///                           (SIZE (1..ub-organizational-unit-name)) }
+            /// ```
+            case .RDNAttributeType.commonName,
+                    .RDNAttributeType.localityName,
+                    .RDNAttributeType.stateOrProvinceName,
+                    .RDNAttributeType.organizationName,
+                    .RDNAttributeType.organizationalUnitName:
+                do {
+                    value = try .init(utf8String: String(ASN1UTF8String(asn1Any: anyValue)))
```

Comment:
This is unnecessary: we can use the tags to identify which case we should hit.

Reply:
Smart idea! Done in https://github.com/apple/swift-certificates/pull/104/commits/17f7c2c1a5ae720665ca41f6c2fef1957deb8777